src/test/java/com/mentorship/userservice/repository/AuthRepositoryTest.java

```diff
@@ -0,0 +1,96 @@
+package com.mentorship.userservice.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.mentorship.userservice.domain.Auth;
+import com.mentorship.userservice.domain.User;
+import com.mentorship.userservice.dto.enums.UserRole;
+import com.mentorship.userservice.repositories.UserRepository;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+@DataJpaTest
+public class AuthRepositoryTest {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    private User user1;
+    private String user1Email;
+    private String user1Name;
+    private String user1LastName;
+    private String user1PhoneNumber;
+    private String user2Name;
+
+
+    @BeforeEach
+    public void createUsers() {
+
+        user1Email = "ivan@gmail.com";
```

Comment:
i think u can set this values explicitly when you describe field:
` private String user1Email ="ivan@gmail.com"`
and it seems to me that it is better to rename these field to userIvanEmail, userTarasEmail etc

Do u expect that these values will be changed? why don't u use constants?

Reply:
Will fix to constant